libwasmvm/bindings.h

```diff
@@ -342,6 +342,26 @@ typedef struct GoQuerier {
   struct Querier_vtable vtable;
 } GoQuerier;
 
+typedef struct GasReport {
+  /**
+   * The original limit the instance was created with
+   */
+  uint64_t limit;
+  /**
+   * The remaining gas that can be spend
+   */
+  uint64_t remaining;
+  /**
+   * The amount of gas that was spend and metered externally in operations triggered by this instance
+   */
+  uint64_t used_externally;
+  /**
+   * The amount of gas that was spend and metered internally (i.e. by executing Wasm and calling
+   * API methods which are not metered externally)
+   */
+  uint64_t used_internally;
+} GasReport;
```

Comment:
There is a required copy of that file that you can update with `make update-bindings` in the repo root

Reply:
Ah, I see. But don't I need to add the finished build of libwasmvm then?